tsconfig.json

```diff
@@ -78,7 +78,7 @@
     "src/components/VErrorSection/VErrorImage.vue",
     "src/components/VFilters/VLicenseExplanation.vue",
     "src/components/VHeader/VFilterButton.vue",
-    "src/components/VHeader/VSearchBar/**.vue",
+    //    "src/components/VHeader/VSearchBar/**.vue",
```

Comment:
```suggestion
    //    "src/components/VHeader/VSearchBar/**.vue",
```

Comments in json 😢

Reply:
I didn't even realize that this is a `json` file!